client/app/license/spla/add/license-spla-add.html

```diff
@@ -1,123 +1,118 @@
-<div data-ng-controller="LicenseSplaAddCtrl">
-
-    <div data-wizard
-         data-wizard-on-cancel="resetAction"
-         data-wizard-on-finish="addSpla"
-         data-wizard-title=":: 'license_spla_add_title' | translate"
-         data-wizard-bread-crumb
-         data-wizard-title-icon="'icon-add-licenses'">
-
-        <div data-wizard-step
-             data-wizard-step-valid="model.selected.server"
-             data-wizard-step-on-load="load">
-
-            <div class="text-center"
-                 data-ng-show="model.options.availableServers == null">
-                <oui-spinner></oui-spinner>
-            </div>
-
-            <div data-ng-show="model.options.availableServers.length">
-                <p class="font-weight-bold"
-                   data-translate="license_spla_add_step1_question">
-                </p>
-                <div class="form-group">
-                    <label for="availableServers"
-                           class="control-label"
-                           data-translate="license_spla_add_step1_server">
-                    </label>
-                    <select class="form-control"
-                            id="availableServers"
-                            name="availableServers"
-                            data-ng-options="type for type in model.options.availableServers"
-                            data-ng-model="model.selected.server">
-                        <option selected="selected"
-                                value=""
-                                data-ng-disabled="true"
-                                data-translate="license_spla_add_step1_server_placeholder">
-                        </option>
-                    </select>
-                </div>
-            </div>
+<div data-wizard
+     data-wizard-on-cancel="$ctrl.endAction.bind($ctrl)"
+     data-wizard-on-finish="$ctrl.addSpla.bind($ctrl)"
+     data-wizard-title=":: 'license_spla_add_title' | translate"
+     data-wizard-bread-crumb>
+
+    <div data-wizard-step
+         data-wizard-step-valid="$ctrl.selected.server"
+         data-wizard-step-on-load="$ctrl.load.bind($ctrl)">
+
+        <div class="text-center"
+             data-ng-if="!$ctrl.options.availableServers">
+            <oui-spinner></oui-spinner>
+        </div>
 
-            <p class="text-danger"
-               data-ng-show="model.options.availableServers.length != null && !model.options.availableServers.length"
-               data-translate="license_spla_add_step1_server_unavailable">
+        <div data-ng-if="$ctrl.options.availableServers.length">
+            <p class="font-weight-bold"
```

Comment:
No `strong`?

Reply:
Used as an helper and applying margins as well.